src/script/event/EventRepository.js

```diff
@@ -56,12 +59,13 @@ export class EventRepository {
     };
   }
 
+  // TODO: Will be replaced with "EventSource"
```

Comment:
outdated, maybe?

Reply:
No, it will be done during a second refactoring. `EventRepository.SOURCE` will be removed from all parts of the app.

Reply:
Ah ok